### ISSUE TITLE
Failure to evaluate

### ISSUE THREAD

**Issue by HaniaRiim19** [alibaba-edu/simple-effective-text-matching-pytorch#10]
whenever i run evaluate.py it gives all predictions = 0 and thus an accuracy of 50% on the test.txt file

**Comment by hitvoice** [comment]
Could you provide your evaluation command & related config file?

**Comment by HaniaRiim19** [comment]
Hello!
I was able to run it with Pytorch after all! The tensorflow version was
deprecated.

But thank you
